### ISSUE TITLE
DDW IATI data issues tracker - crosscheck?

### ISSUE THREAD

**Issue by matmaxgeds** [codeforIATI/iati-data-bugtracker#15]
Check and add issues from the DI DDW [IATI data issues bugtracker](https://docs.google.com/spreadsheets/d/18_qPilad5LDqSkErqvxOGBqMn0vBfIW4-CNMBbR5ack/edit#gid=0) - or maybe these are not appropriate here as are mainly data quality issues, not data publishing issues?

**Comment by andylolz** [comment]
Good point. Data quality issues are also welcome here! It might be good to clarify this in the README, in fact.

**Comment by matmaxgeds** [comment]
Cool, I will also see if I can think of a way to setup a public interface for these issues e.g. pulling from the github API so that there is a page (iati-data-issues-codeforiati.org etc) that anyone can go to to check if the data they are using has any current/outstanding issues that they should consider, for most of my users, that is the biggest hurdle e.g. benefiting from the crowd knowledge that e.g. German disbursements are cumulative pre 2010 and similar. In a dream scenario, when someone got a download from the datastore, they could ask for the community metadata that would look at the publishers in their download, and then suck any applicable issues.......

Which is a long way of saying, maybe in this issues tracker, we can use the labels to mark the issue as 'publishing issue' or 'data quality' etc - even better if we can find a way to add a label for the publisher_ID, so that the API could link it up to the data it applies to.

I will add this as a IATI_idea in a moment

**Comment by andylolz** [comment]
Really good ideas here – thanks @matmaxgeds!

We can definitely improve the labelling on this repo – I think that’s a good first step.

I also like the idea of capturing publisher and/or dataset IDs in the labels, so they can be consumed elsewhere.

Another thing (only tangentially related really) about this repo is: it would be nice to get a reminder “is this still an issue?” nudge, to encourage the issue creator to check back for any changes. I wonder if we can use [probot-stale](https://github.com/apps/stale) for that.

**Comment by andylolz** [comment]
> Another thing (only tangentially related really) about this repo is: it would be nice to get a reminder “is this still an issue?” nudge, to encourage the issue creator to check back for any changes. I wonder if we can use probot-stale for that.

^^ I’ve added this to this repo. Let’s see how it goes!
